### ISSUE TITLE
HDMI VRR not working on any display

### ISSUE THREAD

**Issue by toriko2** [IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT#590]
### Checklist [README]

- [X] Device is not undervolted nor overclocked
- [X] Device is using the [latest drivers](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/wiki/Intel-driver-updates-and-changelogs)
- [X] Application is not cracked, modded and use the latest patch

### Application [Required]

Windows

### Processor / Processor Number [Required]

AMD 7700x

### Graphic Card [Required]

ASRock Intel Arc A750

### GPU Driver Version [Required]

31.0.101.4952

### Rendering API [Required]

- [ ] Vulkan
- [ ] OpenGL
- [ ] DirectX12
- [ ] DirectX11
- [ ] DirectX10
- [ ] DirectX9
- [X] Not applicable

### Windows Build Number [Required]

- [X] Windows 11 22H2
- [ ] Windows 11 21H2
- [ ] Windows 10 22H2
- [ ] Windows 10 21H2
- [ ] Other (Please specify)

### Intel System Support Utility report

[igcit_ssu.txt](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/files/13298384/igcit_ssu.txt)

### Description and steps to reproduce [Required]

After testing my ASRock Arc A750 on multiple displays, using either HDMI port, I was unable to activate VRR from either Arc Control, or the Intel Command Center. This was tested primarily on an LG CX, with both AMD fressync premium enabled and disabled from within the TV's settings. DisplayPort VRR works fine though.

**Comment by Karen-Intel** [comment]
@toriko2 can you share your display specs? To find out if this behavior has been spotted with similar HW
Also, please include your display settings in Windows

Karen

**Comment by toriko2** [comment]
Hello Karen,

The displays I tested this on are the LG CX, Samsung QN90B, and a Koorui 24E3. 
Within windows I tested a number of display settings:
For the LG CX and Samsung QN90B I tried:
4k/120Hz, 4k/60, 1080p/120, and 1080p/60

On the Koorui 24E3 I tried:
1080p/144Hz, 1080p/120, and 1080p/60

With all of these configs the VRR option in Arc Control listed as "Not supported" and in the command center the adaptive sync option was greyed out. Both HDMI ports were tested.

Also I confirmed these displays supported VRR by testing them on an AMD card where each display worked correctly.

**Comment by Karen-Intel** [comment]
Okido @toriko2 

I will find out if we have seen the same issue in other displays or if we have similar models to try with HDMI, because I have a DP to a mini DP port and it is enabled for me in ARC Control

![image](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/assets/91162218/610d7a55-ffb3-4a83-82c7-99668843c5ed)

Keep you posted

Karen

**Comment by toriko2** [comment]
Yep DisplayPort works perfectly for me as well! Just HDMI where I'm running into these issues.

**Comment by Susie1818** [comment]
Only AMD's proprietary Freesync supports HDMI 2.0 ports. Generic VESA Adaptive-Sync requires HDMI 2.1 to run. If your monitors don't have HDMI 2.1 ports, VRR cannot be established because HDMI 2.0 technically doesn't include such protocol for VRR.

On the other hand, DisplayPort has supported VRR since a quite early version 1.2a.

**Comment by toriko2** [comment]
Hi Susie appreciate the response! However two of the displays I tested previously (LG CX and Samsung QN90B), both support HDMI 2.1 and the problem still occurred.

**Comment by freak2fast4u** [comment]
> Only AMD's proprietary FreeSync supports HDMI 2.0/1.4 ports. Generic VESA Adaptive-Sync requires HDMI 2.1 to run. If your monitors don't have HDMI 2.1 ports, VRR cannot be established because HDMI 2.0/1.4 technically doesn't include such protocol for VRR.

AMD FreeSync is AMD's implementation of VESA's adaptative sync IIRC, and it's open by design. I own a Samsung TV that is both HDMI 2.0 and HDMI-VRR compatible. Mind you, I haven't tried plugging my A770 into my TV just yet ... will keep you updated someday if I do.

Quote from : https://en.wikipedia.org/wiki/FreeSync

![image](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/assets/3486369/5a0c335a-03b7-49bc-a46d-798de125fcf4)

**Comment by kpsam2000Intel** [comment]
1. VESA Adaptive Sync is just for DispayPort. For HDMI it's called VRR.
2. HDMI2.1 VRR as per spec is from 2.1 onwards though there is no technical way to know whether a HDMI panel is 2.0 or 2.1. So HDMI VRR capability as per spec is discovered from panel's EDID (capability) blocks. If a panel (including FreeSync) represent the right HDMI VRR EDID indications for VRR, then Intel GPU will allow VRR option for that panel.

**Comment by Karen-Intel** [comment]
Hey @toriko2 any update on this thread?

**Comment by toriko2** [comment]
Hey @Karen-Intel I retested my displays on the latest driver (5186) and found no change in VRR suppport. I was actually waiting for you to update me, as on your previous comment you mentioned seeing if other displays had the same issue. No one else in this thread has tested any displays, but if anyone here can that'd be cool as well.

**Comment by toriko2** [comment]
Also @Karen-Intel I haven't been able to find any real documentation online that the ARC gpus even support VRR over HDMI so perhaps you could ask someone internally if this is even an intended functionality.

**Comment by Karen-Intel** [comment]
@toriko2 You're absolutely right. I'll find out :) Will update this week

**Comment by Karen-Intel** [comment]
Hey @toriko2 I just confirmed.
VRR support was added on 2.1b spec on 2023 and if you check your card specifications you will see you do have 2.0 b
So, it is expected behavior.
![image](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/assets/91162218/0abc46e1-a252-4b51-b302-2065899645be)

Based on [this article ](https://www.hdmi.org/spec21sub/variablerefreshrate) you can see not many GPUs support VRR standard over HDMI.

With this, would you agree for us to close this case? Thank you so much

Karen

**Comment by toriko2** [comment]
![image](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/assets/98915401/700499d3-d7ac-401f-bb07-7a91ac349aae)

This what your looking for?

**Comment by Karen-Intel** [comment]
> ![image](https://private-user-images.githubusercontent.com/98915401/299494924-700499d3-d7ac-401f-bb07-7a91ac349aae.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDYxNDEzNDcsIm5iZiI6MTcwNjE0MTA0NywicGF0aCI6Ii85ODkxNTQwMS8yOTk0OTQ5MjQtNzAwNDk5ZDMtZDdhYy00MDFmLWJiMDctN2E5MWFjMzQ5YWFlLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAxMjUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMTI1VDAwMDQwN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTlhMTk1YmQ5MzQ5MDFiNWMxYjAxNTY4MDdjN2YzMzhiMTg2MDhiZTc3NGZlMGYyYmViMTc2YmM2MmYzMDg3ZjQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.3qaFnB6YD4WTyjS1DmdEH3v_-rEp3RxDxj0-lhAL2F0)
> 
> This what your looking for?

Yes, can you share the other two please? 

Karen

**Comment by toriko2** [comment]
![image](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/assets/98915401/6293dc03-4a1d-4d93-8566-a3fd2c30a077)
![image](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/assets/98915401/a69005df-ad63-4d19-8e5f-82c093b1697a)

Apolgies for the delay. Heres the edid for 1 other display. I also provided extra edid info from another app called custom resolution utility.

Unfortunately I cant get to my other display at the moment as its at another house. Though I could go get the EDID info tomorrow if needed

**Comment by Karen-Intel** [comment]
> ![image](https://private-user-images.githubusercontent.com/98915401/299498541-6293dc03-4a1d-4d93-8566-a3fd2c30a077.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDYxNDI2NDIsIm5iZiI6MTcwNjE0MjM0MiwicGF0aCI6Ii85ODkxNTQwMS8yOTk0OTg1NDEtNjI5M2RjMDMtNGExZC00ZDkzLTg1NjYtYTNmZDJjMzBhMDc3LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAxMjUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMTI1VDAwMjU0MlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTgxYTliYzc2ZDUwNzYwMjgzMjUyNDUyYWU2YTAyNWU1YjM5NmViMzk1NmRkNTQ1NTljY2Y0ODY4OWU1ODY4ZGQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.-ohy3_jjojm2XToFV3nk5uCD24M55Ma-aVD3VU3xM50) ![image](https://private-user-images.githubusercontent.com/98915401/299498601-a69005df-ad63-4d19-8e5f-82c093b1697a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDYxNDI2NDIsIm5iZiI6MTcwNjE0MjM0MiwicGF0aCI6Ii85ODkxNTQwMS8yOTk0OTg2MDEtYTY5MDA1ZGYtYWQ2My00ZDE5LThlNWYtODJjMDkzYjE2OTdhLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAxMjUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMTI1VDAwMjU0MlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTE3MmMxN2ZmODQ5ZGQyYmNmNTkxNDJhZWFkOTE1ZDFhNzg4ZGRiY2FjMjMxZmI0YWM1YmYwN2ZhZmQ2ZjJmZDgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.MgjC_ZFVbtdUjW-2SnmxO3mbMWlQeulIJ8bmmKVVNrU)
> 
> Apolgies for the delay. Heres the edid for 1 other display. I also provided extra edid info from another app called custom resolution utility.
> 
> Unfortunately I cant get to my other display at the moment as its at another house. Though I could go get the EDID info tomorrow if needed

Hey @toriko2 yeah, if you can that would be great, it would give me more info :)

Talk to you tomorrow then 

Karen

**Comment by Karen-Intel** [comment]
Hey @toriko2 ty for the screenshots. I asked and we need something else 
Could you help by sharing the EDID binary files? They are saved as text files 
Please upload them to a shared folder location (Drive) so we can analyze them

I'll be on the lookout :)

Karen

**Comment by toriko2** [comment]
Quick question it seems the binary file is saved as a .bin file instead of .txt. Is this correct? I'm doing this by going to file-> export EDID to binary file inside the monitorInfoView program

**Comment by Karen-Intel** [comment]
> Quick question it seems the binary file is saved as a .bin file instead of .txt. Is this correct? I'm doing this by going to file-> export EDID to binary file inside the monitorInfoView program

@toriko2 yes, binary files are .bin :)

**Comment by toriko2** [comment]
https://drive.google.com/drive/folders/1ajw_rw4K0euRdABwOA9-Vnd934rJ_cDu?usp=sharing

Let me know if there's any issue accessing the drive folder

**Comment by Karen-Intel** [comment]
Thank you, just downloaded them. Keep you posted :)

Karen

**Comment by Karen-Intel** [comment]
Hi @toriko2 sorry for the delay.

Can you do a small test for me in your LG monitor?
Please set resolution to 2560x1600@120Hz (Display > Advanced Settings) and let me know if VRR is enabled.

TY in advance! 

Karen

**Comment by toriko2** [comment]
Unfortunately VRR is still isn't available.

**Comment by Karen-Intel** [comment]
@toriko2 thank you for confirming so quick. 
Allow me to create an internal report for this and I will get back to you with the number.

Stay tuned!

Karen

**Comment by Karen-Intel** [comment]
Hi @toriko2 :) Sorry for the delay
Here's the internal report number: 14021623069
As soon as we have an update, I'll let you know!

Karen

**Comment by Karen-Intel** [comment]
Notification on thread: 
After consolidating our debug efforts, this has been identified as a 3rd party issue. Please contact PCON for support.
We will be closing this thread after receiving submitter's acknowledgement. Thank you

Karen

**Comment by toriko2** [comment]
Ok I appreciate the debugging efforts and the fact that that PCON conversion chip for the HDMI 2.1 port is probably incompatible with VRR.  

One last thing though, I know the Arc GPUs have a native HDMI 2.0 port, any chance that could support VRR? Apparently the Xbox One, which has HDMI version 1.4, supports VRR so it does seem doable.

Perhaps I could open a feature request form separately to ask for the feature to be added?

**Comment by freak2fast4u** [comment]
> Notification on thread: After consolidating our debug efforts, this has been identified as a 3rd party issue. Please contact PCON for support. We will be closing this thread after receiving submitter's acknowledgement. Thank you
> 
> Karen

Erm ... excuse-me ? 
We're customers of Intel, not customers of PCON ...

@Karen-Intel : What's the rationale behind this position please ?
As much as I would like you guys to succeed and am willing to help diagnose things, I really can't understand that statement.

**Comment by Karen-Intel** [comment]
@toriko2 yes please go ahead and submit it as a separate thread. We will make sure to submit it internally

@freak2fast4u exactly. It is part of being transparent with our customers to do experiments internally, research and inform our final findings :)

@Arturo-Intel let's close this one out please. Ty all for your understanding

Karen
